### PR TITLE
merge experimental SolaraViz examples into their non-experimental examples

### PR DESCRIPTION
**Pull request by wang-boyu**
- merge experiment schelling and boltzmann wealth into non-experimental examples, now that SolaraViz is not experimental
- update README accordingly

Edit: partly address https://github.com/projectmesa/mesa-examples/issues/154

### CODE REVIEW

File: examples/boltzmann_wealth_model/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-mesa~=2.0
+mesa[viz]~=3.0.0b0
```
Comment (EwoutH) [inline examples/boltzmann_wealth_model/requirements.txt:1]: We will keep updating this one
```suggestion
mesa[viz]>=3.0.0b0
```

File: examples/boltzmann_wealth_model/st_app.py
Comment (EwoutH) [inline examples/boltzmann_wealth_model/st_app.py:1]: Maybe mention in the Readme briefly the Streamlit app and what the difference is.

**Review by EwoutH — APPROVED** [review]
Thanks a lot for this, a much needed cleanup.

Two minor comments, otherwise looks good!

File: examples/boltzmann_wealth_model/st_app.py
Comment (wang-boyu) [reply examples/boltzmann_wealth_model/st_app.py:1]: Done! Not very good at explaining the differences but pls feell free to change.

**Comment by EwoutH** [comment]
Really fancy Streamlit app!

image

**Comment by wang-boyu** [comment]
Yes! I used Streamlit for work a few years ago and really liked it. Sadly it doesn't have support for Jupyter notebook; otherwise we could've give it a try.

**Pull request merged**